src/components/issues/Card.module.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+	padding: 1rem;
+	width: 100%;
+	background: #fff;
+	border: 1px solid #d9d9d9;
+	border-radius: 0.5rem;
+	transition: 0.2s ease-out;
+	display: flex;
+	flex-direction: column;
+	margin: 2rem auto;
+	justify-content: space-between;
+	position: relative;
+}
+
+.card:hover {
+	border-color: #e5e5e5;
+	filter: drop-shadow(0 20px 13px rgba(0, 0, 0, 0.02))
+		drop-shadow(0 8px 5px rgba(0, 0, 0, 0.06));
+}
+
+.card__top {
+	display: flex;
+	align-items: flex-start;
+	gap: 2rem;
+}
+
+.card__top__details {
+	flex: 1;
+	max-width: 75%;
+	width: 100%;
+	overflow-x: clip;
```

Comment:
the `width: 100%` will not do anything here

Reply:
You're using `%` here, that's why it will not work as intended. Block elements already have width as 100% by default. When you set max-width as 75%, it means -> take 75% of the containing block's width, not more than that. 

Here's an example that compares two scenarios, hope it helps clear it up. Adjust the size of the view window to see the difference : https://codepen.io/nashitshayan/pen/WNaQGap

Reply:
But it's not a block -- it's a flex element right?

Reply:
Got it, thank you!